### DISCUSSION TITLE
Problems after Nov 2023 Windows update

### DISCUSSION THREAD

**Discussion by Gomeskj** [GrandOrgue/grandorgue#1721] [Help]
After the latest Windows update, I have huge problems that I have never encountered before. Have been using GrandOrgue since 2009 and I have never seen these issues.

When I press a stop, it turns on while I have my mouse on it and then immediately turns off. This happens for all the buttons on the demo organ except for the blower noise and the set button.  I am getting all the sounds of the stops, and if I press a key on the keyboard on the screen, then when I pull a stop I do get the sound of that pipe for the split second that the stop stays on.

I have no ability to detect any midi signals, so I cannot use Grandorgue at all.

I have tried with jOrgan as well. WIth jOrgan, I have no problems pulling stops, but get no sound at all.

Has anyone had the same problems? I reset my computer with a fresh version of Windows but that did not solve the problem. I have deleted the partition that Windows was on, reinstalled Windows from my USB, reinstalled GrandOrgue and still have the same problems. I even uninstalled GrandOrgue and tried with older versions of GrandOrgue. Same problems. Really don't know what else to do.

**Comment by Jean-Gabriel1982** [comment]
Good morning,
What is your version of GrangOrgue ?

**Comment by Gomeskj** [comment]
> Good morning, What is your version of GrangOrgue ?

Thanks for responding. I used the latest version from the SoundForge website. But, I have tried with a couple of older versions too, and have the same problems.  Right now running 0.3.1.2330.  After reloading many times, I now can run the demo organ and the stops stay on when turned on. But I can't use my midi keyboards to play it.

**Comment by Jean-Gabriel1982** [comment]
Ok try the last version from GitHub

https://github.com/GrandOrgue/grandorgue/releases

Tell us if the problem persists ..

**Comment by Gomeskj** [comment]
> Ok try the last version from GitHub
> 
> https://github.com/GrandOrgue/grandorgue/releases
> 
> Tell us if the problem persists ..

Just now I uninstalled the old version and installed the latest version. The problems with the stops seems to be resolved. Thank you. I think I just have to now figure out the midi problems.

**Comment by Jean-Gabriel1982** [comment]
Ok this is a first good point. 
Regarding your Midi problems, if you have the same problem with jOrgan then GO is not the cause ;-)

**Comment by oleg68** [comment]
Midi settins might be changed between the versions. So you should try to configure your midi devices again.

**Reply by Gomeskj** [reply]
I have tried. I let GO just autodetect, but I think the problems I'm having now are not GO problems but just midi problems. My console does not seems to be transmitting midi signals, and I don't think there are any problems with the midi USB cable, or drivers or settings. So I have now been in touch with the person who built the console so he can check whether midi data is actually being transmitted to the computer.

**Reply by midimusic** [reply]
You are doing things the hard way!! You can easily use midi-ox to detect whether your computer is recieving midi from your console.
csw900

**Reply by Gomeskj** [reply]
I tried Midi-OX and didn't see any signals. But in case I was doing something wrong, I also tried a browser app called midimonitor.com. Again, no signals detected.

**Reply by Gomeskj** [reply]
So I found out the what the problems were, and am posting in case anyone else has these problems. The stops going off immediately after turning them on was the result of me turning off the "midi thru" switch on the Roland UM-1G. The organ not sending signals was due to 2 things: the power supply had failed and was not supplying adequate power to the organ; and, the midi cable itself was faulty. I switched the midi cable to an M-Audio one, and everything is now working. Thanks to everyone who provided suggestions!
